operator/pkg/central/values/translation/translation.go

```diff
@@ -193,10 +193,9 @@ func getCentralComponentValues(c *platform.CentralComponentSpec) *translation.Va
 		cv.AddChild("exposure", &exposure)
 	}
 
-	if c.CentralDBEnabled() {
+	if c.DB != nil {
```

Comment:
I take from this that **the helm chart** is already changed to default to provisioning a central-db if the `db` field is unset?

Reply:
I've changed it to be a bit more robust if I understand its purpose correctly